os_net_config/utils.py

```diff
@@ -43,6 +43,23 @@
 WantedBy=basic.target
 """
 
+# dcb_config service shall be created and enabled so that the various
+# dcb configurations shall be done during reboot as well using
+# dcb_config.py installed in path /usr/bin/os-net-config-dcb
+_DCB_CONFIG_SERVICE_FILE = '/etc/systemd/system/'\
+    'os-net-config-dcb-config.service'
```

Comment:
Are you forced to break this line by a linter?

If not, please don't, it looks ugly :)